engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -268,24 +268,19 @@ class WorkflowActor(val workflowId: WorkflowId,
   }
 
   onTransition {
-    case oldState -> terminalState if terminalState.terminal =>
+    case (oldState, terminalState: WorkflowActorTerminalState) =>
```

Comment:
ToL: This isn't a problem in this PR, but I just noticed how enormous this case block is (30 lines!)